### DISCUSSION TITLE
Creating a dynamic dns server

### DISCUSSION THREAD

**Discussion by thibault-cne** [NLnetLabs/domain#368] [Q&A]
Hi, I was wondering if it's possible to create a dynamic dns server (that follows the RFC 2136) with the library and if yes, how can I implement the zone update ?

**Comment by Philip-NLnetLabs** [comment]
Currently it will hard to implement RFC 2136 because the basic message format in domain cannot represent update messages. The reason is that update uses resource records where RDLENGTH is zero.
